csharp/tests/source/test.d

```diff
@@ -93,6 +108,16 @@ class c1 {
     public c1 refMember;
     public c1[] refArray;
     public s1[] structArray;
+    public Duration durationMember;
+    public Date dateMember;
+    public DateTime dateTimeMember;
+    public SysTime sysTimeMember;
+    public TimeOfDay timeOfDayMember;
+    public Duration[] durationArray;
+    public Date[] dateArray;
+    public DateTime[] dateTimeArray;
+    public SysTime[] sysTimeArray;
+    public TimeOfDay[] timeOfDayArray;
```

Comment:
What happens if someone goes 2-D?

Reply:
Currently unsupported. Laeeth asked me to stop working on Ranges once I completed the 1D work.